README.md

```diff
@@ -171,21 +198,18 @@ use fang::Queue;
 
 ```
 
-`Queue::insert_task` method will insert a task with uniqueness or not it depends on `uniq` method defined in a task.
-If uniq is set to true and the task is already in storage this will return the task in the storage.
+If `uniq` is set to true and the task is already in the storage, it won't be inserted again.
```

Comment:
why dont specify that will return the task in the storage if uniq is set to true ?

Reply:
Done in [add comment](https://github.com/ayrat555/fang/pull/95/commits/824afac3d7472cd725344de57fa0121b04d32837)